packages/block-editor/src/components/block-list/content.scss

```diff
@@ -316,8 +316,9 @@ _::-webkit-full-page-media, _:future, :root [data-has-multi-selection="true"] .b
 	}
 }
 
-.is-focus-mode .block-editor-block-list__block.is-content-locked.has-child-selected,
```

Comment:
Removed this selector, I'm not sure what it was used for. I tested and it doesn't seem to work in trunk.

Reply:
`is-content-locked` doesn't seem to exist as a class so this was probably missed during an earlier refactor?

Reply:
We don't appear to output `is-content-locked` as a classname in `useBlockProps` either, so this removal looks fine to me 👍